packages/web3-eth/src/index.ts

```diff
@@ -41,6 +41,9 @@ along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
  * const web3 = new Web3(Web3.givenProvider || 'ws://some.local-or-remote.node:8546');
  * web3.eth.getBalance('0x407d73d8a49eeb85d32cf465507dd71d507100c1').then(console.log);
  * > 1000000000000n
+ *```
+ *
+ * With `web3-eth` you can also subscribe (if supported by provider) to events in the Ethereum Blockchain, using the `subscribe` function. See more at the {@link subscribe} function.
```

Comment:
<img width="978" alt="Screenshot 2023-03-17 at 12 15 58 PM" src="https://user-images.githubusercontent.com/29740020/225960469-fa0a3888-cb50-4c7b-845f-144fdd3895fd.png">
the link doesnt seem to be working, would u know why?

Reply:
I didn't use a link, just bolded the text. Would a link be useful (since we are inside the same package)?

Reply:
`See more at the {@link subscribe} function.` {@link subscribe} i think when you use link here, subscribe should redirect to the subscribe function